src/System.Buffers.ReaderWriter/System/Buffers/Writer/BufferWriterT_ints.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers.Text;
+using System.Runtime.CompilerServices;
+
+namespace System.Buffers.Writer
+{
+    public ref partial struct BufferWriter<T> where T : IBufferWriter<byte>
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public void Write(int value, StandardFormat format = default)
+        {
+            int written;
+            while (!Utf8Formatter.TryFormat(value, Buffer, out written, format))
```

Comment:
Just for myself. Why don't you use `out` variables? Is it because of the coding style?

Reply:
You mean why don't I do "out int written"? If yes, then because of scoping. I need written outside of the while loop.

Reply:
Ah, thanks. Didn't know the expression variables scoping is different for loops.